packages/pretty-format/tsconfig.json

```diff
@@ -3,5 +3,9 @@
   "compilerOptions": {
     "rootDir": "src",
     "outDir": "build"
+  },
+  "references": [{
+    "path": "../jest-types"
   }
+  ]
```

Comment:
Looks like we don't lint the ts configs :D

Reply:
heh, fixed